c2corg_ui/static/js/card.js

```diff
@@ -100,13 +93,14 @@ app.CardController.prototype.translate = function(str) {
 
 /**
  * @export
+ * @return {string | undefined}
  */
-app.CardController.prototype.openCardPage = function() {
-  if (!this.disableClick) {
-    window.location.href = app.utils.buildDocumentUrl(
-      app.utils.getDoctype(this.doc['type']),
-      this.doc['document_id'],
-      this.doc['locales'][0]['lang']);
+app.CardController.prototype.createURL = function() {
+  if (window.location.pathname.indexOf('edit') === -1) {
+    return app.utils.buildDocumentUrl(
+            app.utils.getDoctype(this.doc['type']),
+            this.doc['document_id'],
```

Comment:
I tried by adding a @type in the declaration, but then I would have to initialize ALL the properties (as for document in documentService) because if I only do `@type {appx.Document} this.doc; `  then it will tell me `Property xxx never defined on app.CardController.doc`...